core/kernel/interrupt.c

```diff
@@ -42,7 +43,13 @@ int dt_get_irq(const void *fdt, int node)
 	if (!prop)
 		return it_num;
 
-	return itr_chip->dt_get_irq(prop, len);
+	if (type)
```

Comment:
Wouldn't need this if we could depend on `itr_chip->dt_get_irq()` setting `type` and `prio`.

Reply:
Moved to gic.